src/Illuminate/Notifications/Console/stubs/notifications.stub

```diff
@@ -16,8 +16,8 @@ return new class extends Migration
             $table->string('type');
             $table->morphs('notifiable');
             $table->text('data');
-            $table->timestamp('read_at')->nullable();
-            $table->timestamps();
+            $table->dateTime('read_at')->nullable();
```

Comment:
Is the method really named `dateTime` and not `datetime`? Looks strange because we have `datetimes` method already.

Reply:
there is a casing inconsistency that I was going to address after this PR.